Hangfire.Redis.Tests/project.json

```diff
@@ -1,29 +1,23 @@
 {
-  "testRunner": "xunit",
+    "testRunner": "xunit",
 
-  "dependencies": {
-    "NETStandard.Library": "1.6.0",
-    "Newtonsoft.Json": "9.0.1",
-    "Hangfire.Core": "1.6.5",
-    "StackExchange.Redis": "1.1.608",
-    "xunit": "2.2.0-beta2-build3300",
-    "dotnet-test-xunit": "2.2.0-preview2-build1029",
-    "Moq": "4.6.38-alpha",
-    "Hangfire.Redis.StackExchange": "1.0.0"
-  },
+    "dependencies": {
+        "xunit": "2.2.0-beta4-build3444",
+        "dotnet-test-xunit": "2.2.0-preview2-build1029",
+        "Moq": "4.6.38-alpha",
+        "Hangfire.Redis.StackExchange": {"target": "project"}
+    },
 
-  "frameworks": {
-    "netcoreapp1.0": {
-      "dependencies": {
-        "Microsoft.NETCore.App": {
-          "type": "platform",
-          "version": "1.0.0"
+    "frameworks": {
+        "netcoreapp1.1": {
```

Comment:
`appveyor.yml` needs to reflect this as well, in order to build - got broken due to this.

Might be better to test against both versions.